### ISSUE TITLE
[1.6.0-alpha2] Specific "Content migration options" combination can delete all subitems

### ISSUE THREAD

**Issue by unipheye** [JeffDarchuk/SitecoreSidekick#95]
I'm not sure if this is intentional behavior or not but if you use the following setting in the "Content migration options"...

![image](https://user-images.githubusercontent.com/3968457/154236636-f98ded7b-c0c5-4d1b-90f9-ed12fc5c4699.png)
(_Include subitems of selected item_ deselected and _Delete local items not in remote server (removes items)_ selected)

..all local sub items of the parent item that is selected to be synced will be deleted. Even if the local sub items are exactly the same as the source sub items.

**Comment by markgibbons25** [comment]
Yes I noticed this too. Seems like the "Delete local items" needs to be auto disabled if you disable "Include subitems"
